docs/reverse_proxy.md

```diff
@@ -182,11 +182,9 @@ matrix.example.com {
 
 ```
 frontend https
-  bind :::443 v4v6 ssl crt /etc/ssl/haproxy/ strict-sni alpn h2,http/1.1
-  http-request set-header X-Forwarded-Proto https if { ssl_fc }
-  http-request set-header X-Forwarded-Proto http if !{ ssl_fc }
-  http-request set-header X-Forwarded-For %[src]
-
+  # Uncomment the line in the end to disable TLS <= v1.2 
```

Comment:
again, I don't think this is the place to be giving the user lots of different options. Let's make a single recommendation. If advanced users want to do things like restrict TLS versions they can research how to do it themselves.